WEEK-3 FUNCTIONS/09-Sema/equal -reverse.py

```diff
@@ -0,0 +1,30 @@
+"""
+Write a function that controls the given inputs whether they are equal to their reversed order or not.
+
+Example:
+
+Input >>> madam, tacocat, utrecht
+
+Output >>> True, True, False
+"""
+
+
+
+
+
+word=input("please enter a word:")
+def equalreverse(words):
+    newlist=[word for word in words if word==word[::-1]]
```

Comment:
word=input("please enter a word:")
def equalreverse(words):
    newlist=[word for word in words]
    if newlist==newlist[::-1]:
        return True
    else:
        return False
print(equalreverse(word))